nightly/expensive.txt

```diff
@@ -128,6 +128,8 @@ expensive --timeout=600 near-chain near_chain tests::gc::test_gc_pine --features
 expensive --timeout=700 near-chain near_chain tests::gc::test_gc_star_large
 expensive --timeout=700 near-chain near_chain tests::gc::test_gc_star_large --features nightly
 
+expensive integration-tests integration_tests tests::client::block_corruption::check_process_flipped_block_fails
+expensive integration-tests integration_tests tests::client::block_corruption::check_process_flipped_block_fails --features nightly
```

Comment:
How long does it take to run this test? Given nayducks stability I would rather have it in integration tests - if feasible.

Reply:
It runs for 5 mins. Maybe we can run it in separate job per commit. Added as a TODO for a follow-up issue.